### PR TITLE
Support additional CTest arguments

### PR DESCRIPTION
**Pull request by ProGTX**
1. `--additional-ctest-args`
2. Replaces the `--verbose` option
3. Include CTest call in conformance report

### CODE REVIEW

File: run_conformance_tests.py [hunk 1]
```diff
@@ -81,22 +82,19 @@ def handle_args(argv):
         help='The name of the implementation to be displayed in the report.',
         type=str,
         required=True)
-    parser.add_argument(
-        '-V',
-        '--verbose',
-        help='Enable verbose CTest output',
-        type=bool,
-        default=False,
-        required=False)
```
Comment (keryell) [inline run_conformance_tests.py]: Where is this verbose mode gone?

File: run_conformance_tests.py [hunk 2]
```diff
@@ -105,7 +103,6 @@ def generate_cmake_call(cmake_exe, build_system_name, conformance_filter,
     Generates a CMake call based on the input in a form accepted by
     subprocess.call().
     """
-    import shlex
```
Comment (keryell) [inline run_conformance_tests.py]: Quite less obfuscated... :-)

File: run_conformance_tests.py [hunk 3]
```diff
@@ -127,21 +135,15 @@ def subprocess_call(parameter_list):
...
     Runs ctest, overwriting any cached results.
     """
 
     build_system_call = build_system_call.split()
-    ctest_call = [
-        'ctest', '.', '-T', 'Test', '--no-compress-output',
-        '--test-output-size-passed', '0', '--test-output-size-failed', '0'
-    ]
-    if verbose:
-        ctest_call.append('-V')
```
Comment (keryell) [inline run_conformance_tests.py]: I see. You have generalized the verbose case.
I was thinking "verbose" for the Python script first...

Comment (ProGTX) [reply run_conformance_tests.py]: I guess the option was a bit deceiving before, as it referred only to CTest.

File: run_conformance_tests.py (hunk 1 above)
Comment (ProGTX) [reply run_conformance_tests.py]: Replaced by `--additional-ctest-args`, so the caller can pass `-V` there instead

**Pull request merged**